github_connector_odoo/models/odoo_author.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2016-Today: Odoo Community Association (OCA)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields, api
+
+
+class OdooAuthor(models.Model):
```

Comment:
I don't think so. it's like odoo.module name. it's ok, because all core model are prefixed by ir. (like ir.module.module, ...)